nft/nfts.yaml

```diff
@@ -1144,6 +1145,22 @@ components:
             - AIRDROPS
           default: [SPAM]
       in: query
+    spamConfidenceLevel:
+      name: spamConfidenceLevel
+      description: |-
+        Enum - the confidence level at which to filter spam at.
+        The confidence level set means that any spam that is at that confidence level or higher will be filtered out.
+        For example, if the confidence level is HIGH, contracts that we have HIGH or VERY_HIGH confidence in being spam will be 
+        filtered out from the response.
```

Comment:
Let's add back in this:
```
Confidence Levels:
          - VERY_HIGH
          - HIGH
          - MEDIUM
          - LOW
         ```
Reason being otherwise the user would have to click on the box to see the dropdown list vs. being able to see it right away.